lib/index.js

```diff
@@ -198,8 +198,14 @@ module.exports = class DockerComposeParser {
    */
   static _populateEnvsWithHostFromServices (services) {
     const hostnames = DockerComposeParser._getHostnamesForServices(services)
+    const servicesNameMap = services.reduce((obj, service) => {
+      obj[service.metadata.name] = service.instance.name
+      return obj
+    }, {})
 
     return services.map(service => {
+      service.instance.aliases = Parser.aliasInstanceNameReplacer({ aliases: service.instance._aliases, servicesNameMap })
```

Comment:
Is there a reason why you used _alias and alias, but for env, you just overwrote it?